contracts/core/extensions/CoreAccounting.sol

```diff
@@ -211,7 +211,6 @@ contract CoreAccounting is
         uint[] _quantities
     )
         internal
-        isValidBatchTransaction(_tokenAddresses, _quantities)
```

Comment:
Same ^

Reply:
As I go back...I agree that there's no reason to remove those modifiers. It was breaking before and I thought that was the case...I wonder if something was changed in a rebase cause it definitely shouldn't change anything.